dist/package.json

```diff
@@ -33,10 +33,10 @@
     "firebase": ">=6.3.4 <8"
```

Comment:
```suggestion
    "firebase": ">=6.3.4 <9"
```

Reply:
We need to update the `package.json` file in the `dist` directory because that's what actually gets published to npm. To update it, you can either accept my change, or run the following from the root of the directory:

```bash
npm run clean
npm i
npm run build
```